src/interface/src/app/frontend-constants.ts

```diff
@@ -0,0 +1,6 @@
+export const FrontendConstants = {
```

Comment:
thoughts on putting these in `types/map.types.ts` instead? The default map settings are already there so it feels like perhaps these constants should be as well.

Reply:
Good idea.  I will do that.